beets/ui/__init__.py

```diff
@@ -1253,22 +1253,15 @@ def show_path_changes(path_changes):
         # Print every change over two lines
         for source, dest in zip(sources, destinations):
             color_source, color_dest = colordiff(source, dest)
-            print_("{0} \n  -> {1}".format(color_source, color_dest))
+            print_(f"{color_source} \n  -> {color_dest}")
     else:
         # Print every change on a single line, and add a header
-        title_pad = max_width - len("Source ") + len(" -> ")
-
-        print_("Source {0} Destination".format(" " * title_pad))
+        source = "Source "
+        print_(f"{source:<{max_width}}     Destination")
         for source, dest in zip(sources, destinations):
-            pad = max_width - len(source)
             color_source, color_dest = colordiff(source, dest)
-            print_(
-                "{0} {1} -> {2}".format(
-                    color_source,
-                    " " * pad,
-                    color_dest,
-                )
-            )
+            width = max_width - len(source) + len(color_source)
+            print_(f"{color_source:<{width}}  -> {color_dest}")
 
```

Comment:
I spent like 5 minutes staring at this trying to figure out if it's equivalent or not. The use of the padding and weird spacing could be clearer I think in your version. The previous way isn't very idiomatic in Python terms but your code looks weird in that it has a bunch of spaces in strange places. I have no idea why, looking at the new code, L1260 needs 5 spaces, rather than 4 or 6. I think the clarity here could be improved.

Reply:
I wasn't sure whether anything relied on the exact amount of whitespace output by this, so I tried to preserve the (slightly excessive) whitespace exactly as I found it here.  The main problem is that we want to pad `color_source`, but it's hard to do so because the built-in padding code would mistakenly count the control codes in it too.  If it's okay, I'll just remove the excess whitespace here.

Reply:
Okay, I've fixed this.  The UI will no longer emit excessive whitespace here.